Mathlib/Data/Finset/Lattice/Fold.lean

```diff
@@ -783,6 +791,11 @@ theorem sup'_product_right {t : Finset γ} (h : (s ×ˢ t).Nonempty) (f : β ×
     (s ×ˢ t).sup' h f = t.sup' h.snd fun i' => s.sup' h.fst fun i => f ⟨i, i'⟩ := by
   rw [sup'_product_left, Finset.sup'_comm]
 
+set_option linter.docPrime false in
+theorem isLUB_sup' {α : Type} [SemilatticeSup α] {s : Finset α} (hs : s.Nonempty) :
```

Comment:
This should be `Type*` or better yet
```suggestion
theorem isLUB_sup' {s : Finset α} (hs : s.Nonempty) :
```
since I think there is a `variable {α : Type*} [SemilatticeSup α]` above?

I'd also be inclined to put this result immediately below `le_sup'` and `sup'_le`, same throughout.